scalikejdbc-play-plugin/src/main/scala/scalikejdbc/DBSessionRequest.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015 scalikejdbc.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package scalikejdbc
+
+import play.api.mvc.{ Request, WrappedRequest }
+
+case class DBSessionRequest[A](val dbSession: DBSession, request: Request[A]) extends WrappedRequest[A](request)
```

Comment:
Oops, I forgot to fix when it was changed to case class.
I'll fix and I'll change request's scope to private